app/package.json

```diff
@@ -4,10 +4,17 @@
   "identifier": "com.example.electron-boilerplate",
   "description": "Starter for your Electron application. Out of the box ready for serious stuff.",
   "version": "0.1.0",
+  "build": "1",
```

Comment:
Could you please explain why this `build` property is needed? Why ordinary version can't be used?